### PR TITLE
MVP-100 Breadcrumbs Component

### PR DESCRIPTION
**Pull request by SDToews**
## 🎯 Summary

  
[MVP-100](https://citz-imb.atlassian.net/browse/MVP-100)

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

The deleted files named index.tsx should have been included in the component refactor pull request.

### CODE REVIEW

File: frontend/src/components/Breadcrumbs/makeCrumbs.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { CrumbProperty } from './types';
+
+export const makeCrumbs = (pathname: string, labels: string): CrumbProperty[] => {
```
Comment (BradyMitch) [inline frontend/src/components/Breadcrumbs/makeCrumbs.ts:3]: Could you add a comment explaining this function

File: frontend/src/constants/packages/package.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { IconProps } from 'components';
```
Comment (BradyMitch) [inline frontend/src/constants/packages/package.ts:1]: Remove this file. It was replaced by `types.ts`

File: frontend/src/components/Breadcrumbs/makeCrumbs.ts (hunk 1 above)
Comment (SDToews) [reply frontend/src/components/Breadcrumbs/makeCrumbs.ts:3]: done

**Review by BradyMitch — APPROVED** [review]
Great work!

**Pull request merged**